spec/javascript/e2e/ApplicationNewPage.e2e.js

```diff
@@ -303,7 +303,6 @@ describe('ApplicationNewPage', () => {
     expect(values).toContain(TRUNCATED_FIRST_NAME)
     expect(values).toContain(TRUNCATED_LAST_NAME)
     expect(values).toContain(DATE_OF_BIRTH)
-    expect(values).toContain('Vision impairments;Mobility impairments;Hearing impairments')
```

Comment:
why did this get removed?

Reply:
Case I should have added it here in the first place. It's a sale listing test.